gnirehtet.cmd

```diff
@@ -0,0 +1,150 @@
+@ECHO OFF
+CALL :locate_file ADB %~dp0adb.exe "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
+CALL :locate_file JAVA "%JAVA_HOME%\bin\java.exe"
```

Comment:
Yes, %JAVA_HOME% is the directory where java installed. There is no system var for Java binary in Windows.